editor/import/resource_importer_texture_atlas.cpp

```diff
@@ -325,7 +327,13 @@ Error ResourceImporterTextureAtlas::import_group_file(const String &p_group_file
 			atlas_texture.instance();
 			atlas_texture->set_atlas(cache);
 			atlas_texture->set_region(Rect2(offset, pack_data.region.size));
-			atlas_texture->set_margin(Rect2(pack_data.region.position, Size2(pack_data.image->get_width(), pack_data.image->get_height()) - pack_data.region.size));
+
+			if (pack_data.is_cropped) {
+				atlas_texture->set_margin(Rect2(Vector2(0, 0), Vector2(0, 0)));
+			}
+			else {
+				atlas_texture->set_margin(Rect2(pack_data.region.position, Size2(pack_data.image->get_width(), pack_data.image->get_height()) - pack_data.region.size));
+			}
```

Comment:
Just some nitpicking:
It can be simplified, margin is a zero rect by default:
```suggestion
			if (!pack_data.is_cropped) {
				atlas_texture->set_margin(Rect2(pack_data.region.position, pack_data.image->getsize() - pack_data.region.size));
			}
```

But if it's desired for clarity then it could be:
```suggestion
			if (pack_data.is_cropped) {
				atlas_texture->set_margin(Rect2());
			} else {
				atlas_texture->set_margin(Rect2(pack_data.region.position, pack_data.image->getsize() - pack_data.region.size));
			}
```
or:
```suggestion
			atlas_texture->set_margin(pack_data.is_cropped ? Rect2() : Rect2(pack_data.region.position, pack_data.image->getsize() - pack_data.region.size));
```

Reply:
@boruok I see it's your first PR, congrats! :tada: Because you've clicked that "auto add proposed changes button" (or however it's labelled) now you have a second commit added to the PR. A single commit PRs are preferred so don't forget to squash these commits ([here's how to](https://docs.godotengine.org/en/3.4/community/contributing/pr_workflow.html#the-interactive-rebase)).

Reply:
Also I made a typo in my suggestions above (`getsize()` instead of `get_size()`) so you need to fix that. :wink: